code/modules/reagents/reagent_dispenser.dm

```diff
@@ -76,6 +76,14 @@
 		..()
 		reagents.add_reagent("water",1000)
 
+/obj/structure/reagent_dispensers/watertank/high
+	name = "high-capacity water tank"
+	desc = "A highly-pressurized water tank made to hold vast amounts of water.."
+	icon_state = "watertank_high"
+	New()

```

Comment:
Gonna nitpick and ask you not use relative pathing, instead using ``/obj/structure/reagent_dispensers/watertank/high/New()``, since our standards page says so.